shared-bindings/qrio/QRInfo.c

```diff
@@ -63,3 +63,39 @@ const mp_obj_namedtuple_type_t qrio_qrinfo_type_obj = {
         MP_QSTR_data_type,
     },
 };
+
+//| class QRPosition:
+//|     """Information about a non-decoded QR code"""
```

Comment:
Why didn't you add to the Info class?

Reply:
Because there is no overlap between the two sets of information. I either want one or the other, and a named tuple can't have empty fields.